products/sle/main.pm

```diff
@@ -463,7 +466,7 @@ sub load_inst_tests() {
         loadtest "installation/start_install.pm";
     }
     loadtest "installation/install_and_reboot.pm";
-    if (check_var('BACKEND', 'svirt')) {
+    if (check_var('BACKEND', 'svirt') and check_var('ARCH', 's390x')) {
```

Comment:
see the comment on the next line... 
though it should be changed to "on s390 we need..."